Resources/input/fsh/ISiKBookOperation.fsh

```diff
@@ -19,14 +19,14 @@ Usage: #example
   * use = #in
   * min = 1
   * max = "1"
-  * documentation	= "Eine Appointment-Ressource entsprechend dem dazugehörigen ISiK-Profil MUSS übergeben werden. Falls das Parameter-Element 'Appointment' als einziges Parameter-Element verwendet wird, SOLL die Appointment-Ressource direkt im HTTP Body übergeben werden, alternativ zu einer Parameter-Ressource inklusive dem eingebetteten Appointment-Element. Invalide Ressourcen MÜSSEN abgelehnt werden durch das Termin Repository (zur Form der Ablehnung s.u.). Das Appointment MUSS den Status 'proposed' enthalten."
+  * documentation	= "Eine Appointment-Ressource entsprechend dem dazugehörigen ISiK-Profil MUSS übergeben werden. Falls das Parameter-Element 'Appointment' als einziges Parameter-Element verwendet wird, SOLL die Appointment-Ressource direkt im HTTP Body übergeben werden, alternativ zu einer Parameter-Ressource inklusive dem eingebetteten Appointment-Element. Invalide Ressourcen MÜSSEN abgelehnt werden durch das Termin Repository (zur Form der Ablehnung s.u.). Das Appointment MUSS den Status 'proposed' enthalten. Bei Terminverschiebungen (keine Terminabsage!) kann ein angepasstes Appointment in diesem Parameter übergeben werden. Appointment-Ressourcen die dem Termin-Repository bekannt sind unter der im Appointment enthaltenen id MÜSSEN durch das Termin-Repository geupdatet werden unter Beibehaltung der id."
   * type = #Appointment
 * parameter[+]
   * name = #cancelled-appt-id
   * use = #in
   * min = 0
   * max = "1"
-  * documentation	= "Für die vorliegende Spezifikation ist die Verschiebung eines Termin eins zwei-stufiger Prozess, bei dem ein Termin storniert und ein neuer Termin neu gebucht wird. Dieser Parameter repräsentiert die Ressourcen-Id des stornierten Appointments. Der uri-Parameter kann eine absoulte URL enthalten, Server SOLLEN jedoch nur Termine für ihre eigene Domäne verwalten. Im neu-angelegten Appointment MUSS eine Reference auf den abgesagten Termin hinterlegt werden (vgl. Appointment.extension:replaces). Der Status der abgesagten Ressource MUSS durch den Server angepasst werden."
+  * documentation	= "Für die vorliegende Spezifikation ist die Absage eines Termin ein zwei-stufiger Prozess, bei dem ein Termin storniert und ein neuer Termin neu gebucht wird. Dieser Parameter repräsentiert die Ressourcen-Id des stornierten Appointments. Der uri-Parameter kann eine absoulte URL enthalten, Server SOLLEN jedoch nur Termine für ihre eigene Domäne verwalten. Im neu-angelegten Appointment MUSS eine Reference auf den abgesagten Termin hinterlegt werden (vgl. Appointment.extension:replaces). Der Status der abgesagten Ressource MUSS durch den Server angepasst werden."
```

Comment:
question: geht es hier wirklich um die "Absage"? Die Absage ist doch laut Spec mit dem PATCH des Status-Feldes möglich? M.m.n. passen die Sätze danach nicht: zweistufiger Prozess, "neu-angelegte Appointment" etc. Es geht m.M.n. um Verschiebung?

Reply:
Ja, als Client hat man hier zwei Möglichkeiten. Wenn ich die 2 in 1 Option mit der Operation haben möchte als Client, dann kann man die Operation verwenden. Nur eine reine Absage ginge auch per PATCH.

Reply:
@alexzautke wenn ich es richtig sehe, erfordert die Parameters-Ressource auf die $book-Operation zwingenderweise eine appt-resource-Angabe (Kardinalität 1:1). Was schreibe ich da rein, wenn ich eine reine Absage des unter cancelled-appt-id-angegebenen Termins machen möchte? Den abzusagenden Termin und gebe ihn dadurch quasi doppelt an (in appt-resource und in cancelled-appt-id)?

Die letzten Sätze zu cancalled-appt-id (zweistufiger Prozess, neu angelegte Appointment) machen m.M.n. weiterhin nur Sinn für das Verschieben-Szenario, nicht für das Absagen-Szenario...

Reply:
Neuer Versuch unter https://github.com/gematik/spec-ISiK-Terminplanung/pull/177